include/RAJA/policy/cuda/kernel/Sync.hpp

```diff
@@ -54,11 +54,13 @@ namespace statement
 struct CudaSyncThreads : public internal::Statement<camp::nil> {
 };
 
+#if __CUDACC_VER_MAJOR__ >= 9
```

Comment:
This should be #if CUDART_VERSION >= 9000

Reply:
Good catch. Should work now for the CUDA-clang build. As discussed, this is the only place in kernel where __syncwarp() is used.